File: leet_0378_kth_smallest_sorted___bruteforce.cpp
class Solution {
public:
    int kthSmallest(vector<vector<int>>& m, int k)
    {
        vector<int>     g;
        int             side, i, j;

        i = -1;
        side = (int) m.size();
        while (++i < side)
        {
            j = -1;
            while (++j < side)
                g.push_back(m[i][j]);
        }
        sort(g.begin(), g.end());
        return g[k - 1];
    }
};

File: leet_0378_kth_smallest_sorted___bruteforce.java
class Solution {
    public int kthSmallest(int[][] matrix, int k) {
        int     side = matrix.length, i = -1, j;
        int[]   g = new int[side * side];

        while (++i < side) {
            j = -1;
            while (++j < side)
                g[i * side + j] = matrix[i][j];
        }
        Arrays.sort(g);
        return g[k - 1];
    }
}
